src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionIntroViewModel.swift

```diff
@@ -12,17 +12,17 @@ class ExposureSubmissionIntroViewModel {
 	// MARK: - Init
 	
 	init(
+		onPositiveSelfTestButtonTap: @escaping () -> Void,
+		onSelfReportSubmissionButtonTap: @escaping () -> Void,
```

Comment:
You can use the type alias `CompletionVoid` if you want

Reply:
Would keep this just to be consistent here